src/Symfony/Component/Notifier/Message/SmsMessage.php

```diff
@@ -38,7 +39,15 @@ public function __construct(string $phone, string $subject)
 
     public static function fromNotification(Notification $notification, SmsRecipientInterface $recipient): self
     {
-        return new self($recipient->getPhone(), $notification->getSubject());
+        $message = new self($recipient->getPhone(), $notification->getSubject());
+        $message->notification = $notification;
+
+        return $message;
+    }
+
+    public function getNotification(): ?Notification
```

Comment:
```suggestion
    public function getNotification(): Notification
```

or how can it return null?